Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ruru"
-version = "0.8.1"
+version = "0.9.0"
```

Comment:
Please revert the changes of `Cargo.toml` and rebase the branch on top of `master`. I released a new version of `ruby-sys` and updated this file accordingly.